Core/Frameworks/Baikal/Resources/Templates/Navigation/Topbar/Anonymous.html

```diff
@@ -0,0 +1,20 @@
+{% autoescape false %}
+<div class="navbar navbar-inverse navbar-fixed-top">
+	<div class="navbar-inner">
+		<div class="container">
+			<a class="btn btn-navbar" data-toggle="collapse" data-target=".nav-collapse">
+				<span class="icon-bar"></span>
+				<span class="icon-bar"></span>
+				<span class="icon-bar"></span>
+			</a>
+			<a class="brand" href="{{ homelink }}"><img style="vertical-align: text-top; line-height: 20px;" src="res/core/BaikalAdmin/Templates/Page/baikal-text-20.png" /> Server</a>
+			<div class="nav-collapse">
+				<ul class="nav">
+					<li> <a href="admin/"><i class="icon-eject icon-white" style="transform: rotateX(150deg)"></i> Login</a></li>
+					<li> <a href="http://sabre.io/baikal/"><i class="icon-heart icon-white"></i> Get your own</a></li>
```

Comment:
I think this should open in a new tab

Reply:
and fixed in 0913b887458cff0fb9a6cecf4fa6129bcd6101ab =]